auth/auth.go

```diff
@@ -118,32 +118,21 @@ func NewClient(ctx context.Context, c *internal.AuthConfig) (*Client, error) {
 		return nil, err
 	}
 
+	is, err := identitytoolkit.New(hc)
+	if err != nil {
+		return nil, err
+	}
+
 	return &Client{
 		hc:        &internal.HTTPClient{Client: hc},
+		is:        is,
 		ks:        newHTTPKeySource(googleCertURL, hc),
 		projectID: c.ProjectID,
 		snr:       snr,
-		url:       idToolKitURL,
```

Comment:
Can we also remove url from the definition of `Client` type?

Reply:
Yes of course